doc/whats_new/v0.20.rst

```diff
@@ -1356,4 +1363,4 @@ Mehta, Vinit, Vinod Kumar L, Viraj Mavani, Viraj Navkal, Vivek Kumar, Vlad
 Niculae, vqean3, Vrishank Bhardwaj, vufg, wallygauze, Warut Vijitbenjaronk,
 wdevazelhes, Wenhao Zhang, Wes Barnett, Will, William de Vazelhes, Will
 Rosenfeld, Xin Xiong, Yiming (Paul) Li, ymazari, Yufeng, Zach Griffith, Zé
-Vinícius, Zhenqing Hu, Zhiqing Xiao, Zijie (ZJ) Poh
+Vinícius, Zhenqing Hu, Zhiqing Xiao, Zijie (ZJ) Poh
```

Comment:
Please don't remove this trailing newline

Reply:
Not sure what happened here. My local copy is not missing this line.